statev2/src/interface/wallet_index.rs

```diff
@@ -30,6 +30,15 @@ impl State {
         Ok(wallet_id)
     }
 
+    /// Get the ids of all wallets managed by the local relayer
+    pub fn get_all_wallets(&self) -> Result<Vec<Wallet>, StateError> {
+        let tx = self.db.new_read_tx()?;
+        let wallets = tx.get_all_wallets()?;
+        tx.commit()?;
```

Comment:
Should we be committing universally across the interface?

Reply:
It doesn't matter since it's a read but yeah I'll add that to all these methods so that it's clear what's going on

Reply:
yeah, not necessary for safety, but it is nice to do this for semantic clarity, and fwiw it does seem to have an improvement on storage requirements: https://erthink.github.io/libmdbx/usage.html#:~:text=In%20addition%2C%20as,for%20an%20optimization).